model/src/w3gridmd.F90

```diff
@@ -3316,7 +3308,7 @@ SUBROUTINE W3GRID()
           WRITE (NDSO,2953) CFLTM, WDTHCG, WDTHTH
 #endif
 !
-        WRITE (NDSO,2956) UGBCCFL, UGOBCAUTO, UGOBCDEPTH,TRIM(UGOBCFILE), &
+               WRITE (NDSO,*) UGBCCFL, UGOBCAUTO, UGOBCDEPTH,UGOBCFILE, &
```

Comment:
Can we switch this back to the formatted write statement with the 2956 instead of *?  

This does not get rid of the differences but it does make the differences more obvious in that we went from this being the diffs: 
```
123,125c123,125
<  F T  -10.00000     unset                                                       
<  F F F F F F F F F F T T F F T T         100   1.00000000000000     
<   1.000000013351432E-010  9.999999682655225E-021           0 F
---
>   &UNST UGBCCFL =  T, UGOBCAUTO =  T, UGOBCDEPTH = -10.000, UGOBCFILE=unset,
> ,  EXPFSN =  T,EXPFSPSI =  F,  EXPFSFCT =  F,IMPFSN =  F,EXPTOTAL=  F,  IMPTOTAL=  F,IMPREFRACTION=  F,  IMPFREQSHIFT=  F, IMPSOURCE=  F,  SETUP_APPLY_WLV=  F,  JGS_TERMINATE_MAXITER=  T,  JGS_TERMINATE_DIFFERENCE=  T,  JGS_TERMINATE_NORM=  F,  JGS_LIMITER=  F,  JGS_USE_JACOBI=  T,  JGS_BLOCK_GAUSS_SEIDEL=  T,  JGS_MAXITER=  100,  JGS_PMIN=   1.000,  JGS_DIFF_THR=   0.000,  JGS_NORM_THR=   0.000,  JGS_NLEVEL=  0,  JGS_SOURCE_NONLINEAR=  F
> 
```
to this being the diffs: 
```
123,124c123,124
<   &UNST UGBCCFL =  F, UGOBCAUTO =  T, UGOBCDEPTH = -10.000, UGOBCFILE=unset                                                       ,
< ,  EXPFSN =  F,EXPFSPSI =  F,  EXPFSFCT =  F,IMPFSN =  F,EXPTOTAL=  F,  IMPTOTAL=  F,IMPREFRACTION=  F,  IMPFREQSHIFT=  F, IMPSOURCE=  F,  SETUP_APPLY_WLV=  F,  JGS_TERMINATE_MAXITER=  T,  JGS_TERMINATE_DIFFERENCE=  T,  JGS_TERMINATE_NORM=  F,  JGS_LIMITER=  F,  JGS_USE_JACOBI=  T,  JGS_BLOCK_GAUSS_SEIDEL=  T,  JGS_MAXITER=  100,  JGS_PMIN=   1.000,  JGS_DIFF_THR=   0.000,  JGS_NORM_THR=   0.000,  JGS_NLEVEL=  0,  JGS_SOURCE_NONLINEAR=  F
---
>   &UNST UGBCCFL =  T, UGOBCAUTO =  T, UGOBCDEPTH = -10.000, UGOBCFILE=unset,
> ,  EXPFSN =  T,EXPFSPSI =  F,  EXPFSFCT =  F,IMPFSN =  F,EXPTOTAL=  F,  IMPTOTAL=  F,IMPREFRACTION=  F,  IMPFREQSHIFT=  F, IMPSOURCE=  F,  SETUP_APPLY_WLV=  F,  JGS_TERMINATE_MAXITER=  T,  JGS_TERMINATE_DIFFERENCE=  T,  JGS_TERMINATE_NORM=  F,  JGS_LIMITER=  F,  JGS_USE_JACOBI=  T,  JGS_BLOCK_GAUSS_SEIDEL=  T,  JGS_MAXITER=  100,  JGS_PMIN=   1.000,  JGS_DIFF_THR=   0.000,  JGS_NORM_THR=   0.000,  JGS_NLEVEL=  0,  JGS_SOURCE_NONLINEAR=  F

```
where it's more clear that the true difference is the value of EXPFSN has changed. 

This is one of those diffs that might be helpful to pull out into a separate 1-2 line PR so that we can more clearly see what's truly is answer changing, because here it seems that pulling out the: 
EXPFSN = True->false 
and 
UGBCCFL= True->false is what is causing every single regression test to have an answer changing difference.  If that's one PR it's really easy to say "yes everything is fine" and then this as a second PR, we're just looking at what the other updates are doing.    I can help make this second PR if everyone agrees it would be useful.

Reply:
Hi Jessica, 
I agree, we have put back the formated write. It was for testing purposes put *. 
Cheers
Aron

Reply:
Thanks @aronroland ! 

What are your and @aliabdolali  thoughts on a separate PR for just the change in default values of EXPFSN  and UGBCCFL?   I'd be happy to help make that happen if we all agree it's a good idea.

Reply:
This, can be done. No Problem!